lib/circuitry/configuration.rb

```diff
@@ -26,20 +30,25 @@ def subscribe_async_strategy=(value)
       super
     end
 
+    def subscriber_queue_name=(value)
```

Comment:
I think instead of modifying a different attribute in this method, it'd be more clear if we overrode `subscriber_dead_letter_queue_name`, e.g.:

```ruby
def subscriber_dead_letter_queue_name
  super || "#{subscriber_queue_name}-failures"
end
```